src/slack/slack.js

```diff
@@ -2,6 +2,10 @@ import c from '../config'
 import _knex from 'knex'
 import {createChannel} from 'yacol'
 import moment from 'moment-timezone'
+<<<<<<< HEAD
+=======
+import {makeApiCall} from './slackApi'
+>>>>>>> master
```

Comment:
how is this a resolved conflict? please fix this in next PR. the build crashes currently

Reply:
just remove all this lines

Reply:
this shows 3 problems:
- obviously did not try to run it locally at all, because it would just crash
- did not check the PR after yourself before pushing it to review to someone
- automated tests / linting / compiling either failed and were ignored or we don't have any on the project

please guys make sure you fix all of these problems